index.html

```diff
@@ -132,7 +132,7 @@ <h1 class="header">Individual Rooms</h1>
                         <div class="card-body">
                           <h5 class="card-title text-center semiBold">Zysk</h5>
                           <div class="row d-flex align-items-center justify-content-center">
-                            <a href="https://meet.google.com/mbt-fzzk-dix" class="btn btn-dark px-4 " style="background-color: #006654;"><span class="px-2">WhereBy - Internal</span></a>
+                            <a href="https://whereby.com/zysk-internal" class="btn btn-dark px-4 " style="background-color: #006654;"><span class="px-2">WhereBy - Internal</span></a>
```

Comment:
@PrattDarsh this link is no longer available. Please update this to whereby.com/zysk